test/db/analysis/avr

```diff
@@ -62,6 +62,59 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=test adding ROM maps if file_offset > rom address
+FILE=bins/firmware/arduino_avr.bin
+ARGS=-e asm.cpu=ATmega16 -m 8191
+CMDS=<<EOF
+aaa
+om
+EOF
+EXPECT_ERR=<<EOF
```

Comment:
Use the `REGEXP_FILTER_ERR` to add a regex for the warning, so everything else is ignored.
See https://github.com/rizinorg/rizin/tree/dev/test#commands-tests

Reply:
Can you name a method/idea to match it?
I had tried that for things I could think of like `^WARNING` and all which didn't seem to work

Reply:
So, when I put the whole line:
```
REGEXP_FILTER_ERR=(WARNING: Cannot map the ROM section at 1fff with a size of 4000 due file being mapped at 0x1fff)
EXPECT_ERR=<<EOF
WARNING: Cannot map the ROM section at 1fff with a size of 4000 due file being mapped at 0x1fff
EOF
RUN
```
It says it finds a mismatch.

```

#####################
 1 failed test(s) 🚨
#####################

RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -e asm.cpu=ATmega16 -m 8191 -Qc 'aaa
om
' bins/firmware/arduino_avr.bin
-- stderr
--- expected
+++ actual
@@ -1,1 +1,0 @@
-WARNING: Cannot map the ROM section at 1fff with a size of 4000 due file being mapped at 0x1fff

Wat do?    (f)ix ✅️️️    (i)gnore 🙈    (b)roken ☠️️️    (c)ommands ⌨️    (q)uit 🚪
> 

```
And when I press `f` and fix it, the whole stderr is written back on the file again. And when I try to run rz-test again, this is what I get:

```
#####################
 1 failed test(s) 🚨
#####################

RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -e asm.cpu=ATmega16 -m 8191 -Qc 'aaa
om
' bins/firmware/arduino_avr.bin
-- stderr
--- expected
+++ actual
@@ -1,15 +1,0 @@
-Warning: using oba to load the syminfo from different mapaddress.
[x] Analyze all flags starting with sym. and entry0 (aa))
[x] Analyze function calls (aac)
[x] find and analyze function preludes (aap))
[x] Analyze len bytes of instructions for references (aar))
[x] Check for classess
[x] Finding xrefs in noncode section with analysis.in=io.mapss
[x] Analyze value pointers (aav))
[x] Value from 0x00000000 to 0x00003722 (aav))
[x] 0x00000000-0x00003722 in 0x0-0x3722 (aav))
[x] Emulate functions to find computed references (aaef))
[x] Type matching analysis for all functions (aaft))
[x] Propagate noreturn informationn
[x] Use -AA or aaaa to perform additional experimental analysis..
-WARNING: Cannot map the ROM section at 1fff with a size of 4000 due file being mapped at 0x1fff

Wat do?    (f)ix ✅️️️    (i)gnore 🙈    (b)roken ☠️️️    (c)ommands ⌨️    (q)uit 🚪
> f

``` 

and this never gets fixed when I press `f`.

Reply:
I think the autofixing feature wasn't updated for handling regexps. So this is expected somehow.

Reply:
it's ok. we found another way to do this.